accounts/keystore/keystore_test.go

```diff
@@ -336,7 +336,9 @@ func TestWalletNotifications(t *testing.T) {
 
 	// Shut down the event collector and check events.
 	sub.Unsubscribe()
-	<-updates
+	for ev := range updates {
+		events = append(events, walletEvent{ev, ev.Wallet.Accounts()[0]})
```

Comment:
Hey wait. Can't this be a nil deref in case the `sub` already Erred (and the `ev` is nil?)? 
Guess we'll find out on travis, though

Reply:
I'm not sure, but I think range stops if  the retrived ev is nil